_source/Game.ts

```diff
@@ -23,7 +23,9 @@ class Game {
     static newRun(character: Player): void {
         // if the player character is being used for the first time, unlock its lore note
         let charNote = NotePool.unlockCharacterNote(character);
-        if(charNote) {
+        if (charNote) {
+            // Save unlocked notes to LocalStorage
+            Save.saveNotes();
```

Comment:
Should note-saving calls just be put into the `NotePool.unlockFoo` methods, or should we save them explicitly?